templates/webapi/main/test_result_overview_link.html.ep

```diff
@@ -0,0 +1,5 @@
+<a href="<%= url_for('tests_overview')->query(todo => 1, map { (groupid => $_->{id}) } @$group_list); %>"
```

Comment:
Looks like you've lost the paragraph here. If that's fine then just ignore/resolve the comment. (I'm just noticing it.)

Reply:
I now see myself that having a paragraph makes no sense anymore.